vnext/include/CppWinRTIncludes.h

```diff
@@ -4,22 +4,43 @@
 
 #pragma once
 
+#include <winrt/base.h>
+
 #include <winrt/Windows.Foundation.h>
-#include <winrt/Windows.UI.Composition.h>
-#include <winrt/Windows.UI.Text.h>
-#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+#include <winrt/Windows.UI.Xaml.h>
+
 #include <winrt/Windows.UI.Xaml.Controls.h>
+
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>
+
+#include <winrt/Windows.UI.Xaml.Input.h>
+
+#include <winrt/Windows.UI.Text.h>
+#include <winrt/Windows.UI.Composition.h>
+
 #include <winrt/Windows.UI.Xaml.Hosting.h>
 #include <winrt/Windows.UI.Xaml.h>
 
-namespace winrt {
-using namespace ::winrt::Windows::UI::Xaml;
+#include <winrt/Windows.Globalization.h>
+
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+
+namespace winrt
+{
 using namespace ::winrt::Windows::UI::Core;
-using namespace Windows::Foundation;
-using namespace Windows::Foundation::Collections;
-using namespace Windows::UI::Xaml;
-using namespace Windows::UI::Xaml::Controls;
-using namespace Windows::UI::Xaml::Controls::Primitives;
-using namespace Windows::UI::Composition;
-using namespace Windows::UI::Xaml::Hosting;
-} // namespace winrt
+using namespace ::winrt::Windows::Foundation;
+using namespace ::winrt::Windows::Foundation::Collections;
+using namespace ::winrt::Windows::UI::Xaml;
+using namespace ::winrt::Windows::UI::Xaml::Controls;
+using namespace ::winrt::Windows::UI::Xaml::Controls::Primitives;
+using namespace ::winrt::Windows::UI::Composition;
+
+// using namespace ::winrt::Windows::UI::Xaml::Controls;
+using Control = ::winrt::Windows::UI::Xaml::Controls::Control;
+using ComboBox = ::winrt::Windows::UI::Xaml::Controls::ComboBox;
+using ContentControl = ::winrt::Windows::UI::Xaml::Controls::ContentControl;
+using CheckBox = ::winrt::Windows::UI::Xaml::Controls::CheckBox;
+using ToggleSwitch = ::winrt::Windows::UI::Xaml::Controls::ToggleSwitch;
+}
```

Comment:
I don't think this should be in a header used by all. I realize it's subjective, but at the very least it is controversial and we should talk about it before adding more to it. It would probably be best for you to split this PR up to unblock the auto_revoke fix.

Reply:
Agreed it doesn't seem essential to the PR, but I'm guessing it came in because the revoker types + namespaces were long? This does seem to be the pattern that [WinUI uses](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/inc/CppWinRTIncludes.h), but I'm not sure _why_. I'd love to hear more about that and at least have that captured as a comment.

Reply:
Although I think it has a lot of benefits and put them in header, agree to split this PR

Reply:
WinUI use this pattern because it's easy to switch the namespace from Windows::UI::Xaml to Microsoft::UI::Xaml.
We more or less should do it in the same way in RN windows.
We have duplicated using in every cpp like below, and I think it's better to put all in a header file.

#include <winrt/Windows.UI.Xaml.Controls.h>
...

namespace winrt {
using namespace Windows::Foundation;
using namespace Windows::UI;
using namespace Windows::UI::Xaml;
using namespace Windows::UI::Xaml::Controls;
using namespace Windows::UI::Xaml::Controls::Primitives;
using namespace Windows::UI::Xaml::Media;
} // namespace winrt


---
In reply to: [301723713](https://github.com/microsoft/react-native-windows/pull/2746#discussion_r301723713) [](ancestors = 301723713)

Reply:
Ah. the redirect for MUX versus WUX, that makes sense. A comment in the header here would go a long way towards dispelling any confusion.

Reply:
I rollback the header

---
In reply to: [301730122](https://github.com/microsoft/react-native-windows/pull/2746#discussion_r301730122) [](ancestors = 301730122)